### PR TITLE
Ports sake with new sprites

### PR DESCRIPTION
**Pull request by ohnopigeons**
:cl:
add: You can now get traditional sake and get a first-hand taste of a superior culture. Have I told you about my katana and my trip to-
/:cl:

Because I'm a pedant that wanted to keep and modify certain details, apologies to @DaxDupont 

Renames have new sprites now.

### CODE REVIEW

**Review by ShizCalev — DISMISSED** [review]

**Comment by 81Denton** [comment]
You guys should duel with katanas ingame and the winner gets his PR merged

**Comment by praisenarsie** [comment]
port half-zatochi from tf2

**Comment by sawrge** [comment]
love 2 drink rice juice

**Comment by Armhulen** [comment]
@praisenarsie ok!

**Comment by PKPenguin321** [comment]
Weeb fluid

**Comment by fludd12** [comment]
But, the pun...

**Comment by pigeonsk** [comment]
Which pun?

**Review by Cyberboss — APPROVED** [review]

**Comment by fludd12** [comment]
Tetravulpine Sake => 4 Fox Sake => For Focks Sake

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -298,6 +298,23 @@
...
+/obj/item/reagent_containers/food/drinks/bottle/sake
+	name = "Ryo's traditional sake"
+	desc = "Sweet as can be, and burns like fire going down."
+	icon_state = "sakebottle"
+	list_reagents = list("sake" = 100)
+
+/obj/item/reagent_containers/food/drinks/bottle/sake/Initialize()
+	. = ..()
+	if(prob(10))
+		name = "Fluffy Tail Sake"
```
Comment (Wilchenx) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: fuckup of the year

Comment (pigeonsk) [reply code/modules/food_and_drinks/drinks/drinks/bottle.dm]: my waifu is pure

Comment (fludd12) [reply code/modules/food_and_drinks/drinks/drinks/bottle.dm]: That game is about as pure as Kevin's and me combined

**Pull request merged**